### PR TITLE
Add custom CA settings reference

### LINKED ISSUE #236: Document "Custom CA certificates settings"
**Path to page with content (_do not_ include domain name):**
`/en/os/1.14.x/api/settings/`

**What content is incorrect?**
The settings reference lacks the "[Custom CA certificates settings](https://github.com/bottlerocket-os/bottlerocket#custom-ca-certificates-settings)"

**What should the content say or communicate?**
There is an [ongoing effort](https://github.com/bottlerocket-os/bottlerocket-project-website/issues/203) to move the settings reference currently located on the [Bottlerocket README](https://github.com/bottlerocket-os/bottlerocket) to Bottlerocket.dev

This can be accomplished using the [Settings Documentation DSL](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/METADOC-SETTING-DOCS-DSL.md). Effectively, the settings written out in the README's markdown needs to be converted to this DSL format. You can see an example of how this looks in the [ECS settings reference TOML file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/data/settings/1.14.x/ecs.toml) as well as [the actual ECS content markdown file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/content/en/os/1.14.x/api/settings/ecs/_index.markdown).

The file should be located at `/data/settings/1.14.x/pki.toml`.

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes #236 

**Description of changes:**
Adds settings reference for "Custom CA certificates" (aka `settings.pki.*`)

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: content/en/os/1.14.x/api/settings/pki/_index.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
++++
+title="pki"
+type="docs"
+toc_hide=true
+description="Settings related to Custom CA Certificates (`settings.pki.*`)"
+
++++
+
+By default, Bottlerocket ships with the [Mozilla CA certificate store](https://ccadb.my.salesforce-sites.com/mozilla/IncludedCACertificateReport), but you can add self-signed certificates with `settings.pki.<bundle name>`.
```
Comment (stockholmux) [inline content/en/os/1.14.x/api/settings/pki/_index.markdown:9]: I'd love feedback if the link "Mozilla CA certificate store" is appropriate.

File: content/en/os/1.14.x/api/settings/pki/_index.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+{{% alert title="Tip" color="success" %}}
+If your user data is over the size limit for the platform, you can use `apiclient` with these setting from within a [bootstrap container](https://github.com/bottlerocket-os/bottlerocket#bootstrap-containers-settings) to add certificates.
```
Comment (stockholmux) [inline content/en/os/1.14.x/api/settings/pki/_index.markdown:12]: bootstrap container link should change once #249 is closed.

File: content/en/os/1.14.x/api/settings/pki/_index.markdown (hunk 1 above)
Comment (arnaldo2792) [reply content/en/os/1.14.x/api/settings/pki/_index.markdown:9]: I would prefer either https://curl.se/docs/caextract.html or https://wiki.mozilla.org/CA/Included_Certificates, and let the users navigate to the page your showed. I don't like the advertisement on the `salesforce` website :sweat_smile:

File: content/en/os/1.14.x/api/settings/pki/_index.markdown (hunk 2 above)
Comment (arnaldo2792) [inline content/en/os/1.14.x/api/settings/pki/_index.markdown:12]:
```suggestion
If your user data is over the size limit for the platform, you can use `apiclient` with this setting from within a [bootstrap container](https://github.com/bottlerocket-os/bottlerocket#bootstrap-containers-settings) to add certificates.
```

File: data/settings/1.14.x/pki.toml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+[[docs.ref.pki_bundle-name_data]]
+name_override = "<bundle-name>.data"
+description = "A Base64-encoded PEM-formatted certificate bundle; This setting can contain more than one certificate."
```
Comment (arnaldo2792) [inline data/settings/1.14.x/pki.toml:3]: Not at English expert, but should this be?

```suggestion
description = "A Base64-encoded PEM-formatted certificate bundle; this setting can contain more than one certificate."
```

File: data/settings/1.14.x/pki.toml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+note = """
+Defining a certificate bundle with `data` and but without [`trusted`](#pki_bundle-name_trusted) results in a untrusted bundle.
```
Comment (arnaldo2792) [inline data/settings/1.14.x/pki.toml:5]:
```suggestion
Defining a certificate bundle with `data` but without [`trusted`](#pki_bundle-name_trusted) results in an untrusted bundle.
```

File: data/settings/1.14.x/pki.toml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+data="W3N..."
+"""
+direct_shell = """
+apiclient set \
+  pki.some-bundle.data="W3N..."
+"""
+
+[[docs.ref.pki_bundle-name_trusted]]
+name_override = "<bundle-name>.trusted"
+description = "Defines if the Bottlerocket trusts `<bundle-name>`."
```
Comment (arnaldo2792) [inline data/settings/1.14.x/pki.toml:19]:
```suggestion
description = "Defines if the bundle in `<bundle-name>` should be trusted."
```

File: content/en/os/1.14.x/api/settings/pki/_index.markdown (hunk 1 above)
Comment (stockholmux) [reply content/en/os/1.14.x/api/settings/pki/_index.markdown:9]: Alright, I'll swap it over to the mozilla link. FWIW, I don't think it has anything to do with salesforce, I think that Mozilla just uses salesforce. But you're right - if we are saying Mozilla, we should make it crystal clear this is from them and the link to salesforce's domain confuses that.

**Pull request merged**